client/components/loading-spinner/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Spinner from 'components/spinner';
+
+const LoadingSpinner = ( { inline } ) => {
```

Comment:
I only see two inline `<LoadingSpinner />` - in `PrintLabelDialog` and `StepContainer`. Since `LoadingSpinner` was first introduced here, I thought I'd mention it.

But yeh, can definitely defer to a later PR 👍.